Vuologist/bumpie
# Subcategory test logic

closes #117

Test Instructions:
1. Compile and go to /dashboard
2. Expand a category and hit the + button at the bottom
3. A subcategory should get added with a default name
4. Try to add more than 5 (you should not be able to add more than 5)
